code/modules/surgery/surgery_step.dm

```diff
@@ -9,6 +9,9 @@
 	var/list/chems_needed = list()  //list of chems needed to complete the step. Even on success, the step will have no effect if there aren't the chems required in the mob.
 	var/require_all_chems = TRUE    //any on the list or all on the list?
 	var/silicons_obey_prob = FALSE
+	var/list/preop_sound //Sound played when the step is started
+	var/list/success_sound //Sound played if the step succeeded
+	var/list/falure_sound //Sound played if the step fails
```

Comment:
this should be failure

Reply:
also why are they lists when its always just 1

Reply:
ah, one sec

Reply:
> Op sounds are lists currently in case someone in the future wants to add multiple different sounds, but I can replace them.